console/src/components/Common/Toggle/Toggle.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import Toggle from 'react-toggle';
+import './Toggle.css';
+import 'react-toggle/style.css';
+
+type ToggleProps = {
+  onChange: () => void;
+  icons: boolean;
+  checked: boolean;
+};
+
+const ToggleComponent = (props: ToggleProps) => <Toggle {...props} />;
```

Comment:
Since this component is not doing anything it could be just:
```suggestion
const ToggleComponent = Toggle;
```

Reply:
And if you do this, you don't need typings for props anymore.